### DISCUSSION TITLE
Stream short clips over RTSP without closing stream?

### DISCUSSION THREAD

**Discussion by roger-** [bluenviron/mediamtx#4024] [Questions]
### Question

I have a bunch of short h264 clips that are dynamically and periodically generated and I want to turn them into an RTSP stream. I can use ffmpeg to do this, but I don't want to the stream to close in between the clips (could be minutes or hours). Ideally the last frame of the most recent clip would be repeated but a black frame would be acceptable.

Anyone have an idea how to do this with ffmpeg + mediamtx?

**Comment by aler9** [comment]
Hello,
Currently the server does not support discontinuous streams.
You can generate a continuous stream from segments by using an external software, like OBS or an OpenCV script.
Otherwise, the ability to serve a continuous stream is being tracked here: #2214
